src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenContainer/DragDrop/DropableSpace.scss

```diff
@@ -0,0 +1,19 @@
+.dropable {
+  padding: 8px 0;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+
```

Comment:
remove space :3

Reply:
Nice catch, I will add the sass linter :)